composer.lock

```diff
@@ -3167,6 +3167,5 @@
     },
     "platform-overrides": {
         "php": "7.3.12"
-    },
-    "plugin-api-version": "1.1.0"
+    }
```

Comment:
is that update necessary?

Reply:
Not sure how that happened. I've reverted the changes to composer.lock completely.